packages/driver/src/cy/net-stubbing/add-command.ts

```diff
@@ -279,7 +279,7 @@ export function addCommand (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
         handler = arg2
 
         return {
-          method: matcher,
+          method: matcher.toUpperCase(),
```

Comment:
this doesn't account for the use case `cy.intercept({ method: 'some-lowercase-method' })`, only the shorthand `cy.intercept('some-lowercase-method')` - updated to lowercase on the backend instead